quinn-proto/src/packet.rs

```diff
@@ -18,17 +19,17 @@ use crate::{
 // across QUIC versions), which gives us the destination CID and allows us
 // to inspect the version and packet type (which depends on the version).
 // This information allows us to fully decode and decrypt the packet.
-#[allow(unreachable_pub)] // fuzzing only
+#[allow(unreachable_pub, missing_docs)] // fuzzing only
```

Comment:
If this is no longer only for fuzzing we should get rid of the `missing_docs` and `unreachable_pub` allows.